src/Adapter/Product/ProductDuplicator.php

```diff
@@ -209,6 +221,9 @@ private function duplicateProduct(Product $product): Product
     }
 
     /**
+     * @deprecated since 8.1 and will be removed in next major version.
```

Comment:
The method is private so you don't need to deprecate it you can remove it And also remove the `stringModifier` dependency I believe since it won't be used anymore

Reply:
Done ✅